File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.py
def solution(s):
    result = 0
    i = 0
    while s != "1":
        remove_zero = s.replace("0", "")
        result += len(s) - len(remove_zero)
        s = bin(len(remove_zero))[2:]
        i += 1
    return [i, result]

File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.js
function solution(s) {
    let result = 0;
    i = 0
    while(s !== "1"){
        const removeZero = s.replaceAll("0", "");
        result += s.length - removeZero.length;
        s = removeZero.length.toString(2);
        i++;
    }
    return [i, result];
}